project/Dependencies.scala

```diff
@@ -25,6 +25,7 @@ object Dependencies {
     "com.lihaoyi" %% "os-lib" % "0.9.3", // for writing/compiling C source files
     "org.scala-lang.modules" %% "scala-xml" % "2.2.0",
     "org.scala-lang.modules" %% "scala-parser-combinators" % "2.3.0",
+    "org.scala-lang.modules" %% "scala-java8-compat" % "1.0.2",
```

Comment:
What is this for? This has a NOTICE file so we'll need to update our NOTICE files.